src/php/Config/PhelConfig.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Phel\Config;
+
+use JsonSerializable;
+use Phel\Build\BuildConfig;
+use Phel\Command\CommandConfig;
+use Phel\Filesystem\FilesystemConfig;
+use Phel\Formatter\FormatterConfig;
+use Phel\Interop\InteropConfig;
+
+final class PhelConfig implements JsonSerializable
+{
+    private array $srcDirs = ['src/phel'];
```

Comment:
Not sure if it makes sense to define the default value here (it was already defined in the `phel-config.php` file, isn't it? 😃

Reply:
I think it makes sense to use this as the source of truth for these values. Because the goal (as I understood) is to use this object instead of the raw arrays. 
> TODO: we need to update the docs page: https://phel-lang.org/documentation/configuration/

Reply:
Would you like to create a PR to update the [phel docs page](https://phel-lang.org/documentation/configuration/) regarding this topic, @cgrabenstein? That would be merged after creating the release, but it would be nice to have it ready sooner.

Reply:
Sure, will do!

Reply:
Awesome! One last thing, can you add a line in the `CHANGELOG.md` about this new feature?